<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleChoice</class>
 <widget class="QDialog" name="MultipleChoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>1027</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Answer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Answer_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Answer_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>330</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Answer_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>740</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Answer_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>740</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Answer_6">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>740</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>740</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>740</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>740</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>780</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="result_Label">
   <property name="geometry">
    <rect>
     <x>888</x>
     <y>780</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>770</y>
     <width>1261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Pictures/Saved Pictures/Blue.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>1101</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_Question_4">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;Question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_13">
        <property name="text">
         <string>Potential Answer 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_14">
        <property name="text">
         <string>Potential Answer 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_15">
        <property name="text">
         <string>Potential Answer 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_16">
        <property name="text">
         <string>Potential Answer 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_Question_5">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_17">
        <property name="text">
         <string>Potential Answer 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_18">
        <property name="text">
         <string>Potential Answer 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_19">
        <property name="text">
         <string>Potential Answer 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_20">
        <property name="text">
         <string>Potential Answer 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_Question_6">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_21">
        <property name="text">
         <string>Potential Answer 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_22">
        <property name="text">
         <string>Potential Answer 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_23">
        <property name="text">
         <string>Potential Answer 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_24">
        <property name="text">
         <string>Potential Answer 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1101</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_Question">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_1">
        <property name="text">
         <string>Potential Answer 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_2">
        <property name="text">
         <string>Potential Answer 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_3">
        <property name="text">
         <string>Potential Answer 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_4">
        <property name="text">
         <string>Potential Answer 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_Question_2">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_5">
        <property name="text">
         <string>Potential Answer 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_6">
        <property name="text">
         <string>Potential Answer 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_7">
        <property name="text">
         <string>Potential Answer 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_8">
        <property name="text">
         <string>Potential Answer 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_Question_3">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_9">
        <property name="text">
         <string>Potential Answer 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_10">
        <property name="text">
         <string>Potential Answer 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_11">
        <property name="text">
         <string>Potential Answer 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Potential_Answer_12">
        <property name="text">
         <string>Potential Answer 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>label_7</zorder>
  <zorder>label_Question</zorder>
  <zorder>lineEdit_Answer</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_Question_2</zorder>
  <zorder>label_Question_3</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>label_Question_4</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>label_Question_5</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>label_Question_6</zorder>
  <zorder>lineEdit_Answer_2</zorder>
  <zorder>lineEdit_Answer_3</zorder>
  <zorder>lineEdit_Answer_4</zorder>
  <zorder>lineEdit_Answer_5</zorder>
  <zorder>lineEdit_Answer_6</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>pushButton</zorder>
  <zorder>result_Label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
